recipes/configs/llama2/7B_qat_full.yaml

```diff
@@ -65,7 +65,7 @@ enable_activation_checkpointing: True
 memory_efficient_fsdp_wrap: False
 
 # Reduced precision
-dtype: bf16
+dtype: fp32
```

Comment:
cc @andrewor14

Reply:
This doesn't look right. I will submit a PR to remove that assertion

Reply:
Btw how come this wasn't caught in the tune nightly CI? @joecummings

Reply:
@msaroufim we don't actually test with our "prod" configs, instead we define a set of test configs that we deem to be (pretty) representative of the configs we provide. Unfortunately to do loss parity checks we tend to set `dtype=fp32` in the tests (see [here](https://github.com/pytorch/torchtune/blob/e568b671bf4dd043649611c3d0905967257ccac2/tests/recipes/test_qat_distributed.py#L36) for the QAT test), so as a result this one slipped by